### ISSUE TITLE
All but 2 devices report unavailable

### ISSUE THREAD

**Issue by pwlivewire** [custom-components/alexa_media_player#1842]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**
Devices are constantly unavailable, only 1 device remains on-line. 

i have 8 echo devices all set to UK region and only 1 report available. the rest report unavailable except for a few seconds. 

**To Reproduce**

devices report unavailable all the time

**Expected behavior**
available.

**Screenshots**

**System details**

- Home-assistant (version): Home Assistant 2023.1.5
- alexa_media (version from `const.py` or HA startup): 4.5.3
- alexapy (version from `pip show alexapy` or HA startup):
- Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). : logins work fine, 1 device works fine.

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

logs report 1 error;
[alexapy.alexalogin] Domain amazon.com does not match reported account domain https://www.amazon.co.uk; functionality is not likely to work, please fix
but i've been through each device and it's set to the UK and i've check per your instructions that amazon portal is set to UK. 

**Additional context**

![Alexa](https://user-images.githubusercontent.com/25018910/213585379-896beb55-dc4a-4a0c-93f7-1493c17f1b91.PNG)
![Alexa2](https://user-images.githubusercontent.com/25018910/213585384-33727659-bda4-4cef-b545-dd42d9afedd2.PNG)
![Alexa3](https://user-images.githubusercontent.com/25018910/213585392-eeb6a6f9-cdf9-4794-8040-ce01569caaea.PNG)

**Comment by alandtse** [comment]
> [alexapy.alexalogin] Domain amazon.com does not match reported account domain https://www.amazon.co.uk/; functionality is not likely to work, please fix

Log into amazon.co.uk instead of amazon.com. It's literally in the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ#all-my-media_players-are-unavailable-and-the-component-doesnt-work).

**Comment by pwlivewire** [comment]
your right it is, and i read it.. and it was less than useful. 

"Ensure your Alexa domain is appropriate for your region" 

where do I check it the correct region, on the device? integration? amazon's Alexa device? as I stated on the original post "i've been through each device and it's set to the UK and i've checked per your instructions that the amazon portal is set to UK" I checked what I could and the 2 sentences FAQ gave me nothing to go on.
